README.md

```diff
@@ -1 +1,170 @@
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖Leaderboard <a name="about-project"></a>
+
+
+
+**Leaderboard** is a wonderful  project to see the score of the users.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>HTML ,CSS and JavaScript</summary>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+-  **input fields**
+- **button**
+- **dynamic list**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- Navigate to the location of the folder in your machine:
+
+**`you@your-Pc-name:~$ cd desktop`**
+
+### Prerequisites
+
+In order to run this project you need:
+
+Any popular web browser app, like Chrome, Firefox,etc.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  - Navigate to the location of the folder in your machine:
+
+**`you@your-Pc-name:~$ cd desktop`**
+
+### Install
+
+Install this project with:
+Visual Studio Code and Git
+
+
+### Usage
+
+To run the project, execute the following command:
+
+Right-click on the HTML file
+and choose the browser you wish. 
+
+
+
+### Run tests
+
+To run tests, run the following command:
+
+Right-click on the HTML file
+
+Choose the browser you wish. 
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/MayPyone)
+- Twitter: [@twitterhandle](https://twitter.com/maypyone015)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/may-pyone-9439961a3/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **Remove_buttons**
+- [ ] **Functions_for_completed_activities**
+- [ ] **Animation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project, check my portfolio.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank my learning partners who supported me well thoughout the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
+
```

Comment:
- [ ] [OPTIONAL] Additionally, I encourage you to enhance the license file by replacing the placeholders such as 'fullname,' 'year,' and 'software type.' It's crucial to address this matter seriously. By providing accurate information in the license file, you ensure clarity and legal compliance
  ![image](https://github.com/MayPyone/Leaderboard/assets/104968236/8e1970a5-79e0-44cc-91f0-0b62ba25c4ef)